packages/language-model/src/openai-image.ts

```diff
@@ -46,3 +46,59 @@ export const models = [openaiGptImage1];
 
 export const LanguageModel = OpenAIImageLanguageModel;
 export type LanguageModel = OpenAIImageLanguageModel;
+
+export interface OpenAIImageGenerationParams {
+	width: number;
+	height: number;
+	quality: "auto" | "low" | "medium" | "high";
+}
+
+const openAICostTable: Record<
```

Comment:
@Rindrics We would appreciate it if you could include the URL of the document from which the amounts in openAICostTable are calculated.

Reply:
fixed: https://github.com/giselles-ai/giselle/pull/824/commits/2c0a38c646c19fdd1d6d3903ebd9676d9e1bbbfe